docs/yarn.lock

```diff
@@ -389,6 +389,27 @@
     json5 "^2.1.2"
     semver "^6.3.0"
 
+"@babel/core@^7.17.8":
```

Comment:
I don't see any changes to package.json. What caused this yarn.lock update?

Reply:
I added the gtag plugin to yarn. docusaurus uses yarn install instead of npm

Reply:
where did you add it to yarn? I'm pretty sure yarn uses package.json as well.